src/pudl/package_data/ferc1/dbf_to_xbrl.csv

```diff
@@ -1161,175 +1161,175 @@ f1_co_directors,45,1994,Line 45,,,,
+f1_comp_balance_db,12,2004,(Less) Accum. Prov. for Amort. of Nucl. Fuel Assemblies (120.5),reported_value,accumulated_provision_for_amortization_of_nuclear_fuel_assemblies,,
+f1_comp_balance_db,13,2004,Net Nuclear Fuel (Enter Total of lines 7-11 less 12),calculated_value,nuclear_fuel_net,,
+f1_comp_balance_db,14,2004,Net Utility Plant (Enter Total of lines 6 and 13),calculated_value,utility_plant_and_nuclear_fuel_net,,
+f1_comp_balance_db,15,2004,Utility Plant Adjustments (116),reported_value,other_electric_plant_adjustments,,
+f1_comp_balance_db,16,2004,Gas Stored Underground - Noncurrent (117),reported_value,gas_stored_underground_noncurrent,,
+f1_comp_balance_db,17,2004,OTHER PROPERTY AND INVESTMENTS,header,,,
+f1_comp_balance_db,18,2004,Nonutility Property (121),reported_value,nonutility_property,,
+f1_comp_balance_db,19,2004,(Less) Accum. Prov. for Depr. and Amort. (122),reported_value,accumulated_provision_for_depreciation_and_amortization_of_nonutility_property,,
+f1_comp_balance_db,20,2004,Investments in Associated Companies (123),reported_value,investment_in_associated_companies,,
+f1_comp_balance_db,21,2004,Investment in Subsidiary Companies (123.1),reported_value,investment_in_subsidiary_companies,,
+f1_comp_balance_db,22,2004,"(For Cost of Account 123.1, See Footnote Page 224, line 42)",header,,,
+f1_comp_balance_db,23,2004,Noncurrent Portion of Allowances,reported_value,noncurrent_portion_of_allowances,,This xbrl factoid is mapped to two dbf rows that might have different sign conventions.
+f1_comp_balance_db,24,2004,Other Investments (124),reported_value,other_investments,,
+f1_comp_balance_db,25,2004,Sinking Funds (125),reported_value,sinking_funds,,
+f1_comp_balance_db,26,2004,Depreciation Fund (126),reported_value,depreciation_fund,,
+f1_comp_balance_db,27,2004,Amortization Fund - Federal (127),reported_value,amortization_fund_federal,,
+f1_comp_balance_db,28,2004,Other Special Funds (128),reported_value,other_special_funds,,
+f1_comp_balance_db,29,2004,Special Funds (Non Major Only) (129),reported_value,special_funds_non_major,TRUE,"Close to special_funds, but only includes non major entities"
+f1_comp_balance_db,30,2004,Long-Term Portion of Derivative Assets (175),reported_value,derivative_instrument_assets_long_term,,This xbrl factoid is mapped to two dbf rows that might have different sign conventions.
+f1_comp_balance_db,31,2004,Long-Term Portion of Derivative Assets  Hedges (176),reported_value,derivative_instrument_assets_hedges_long_term,,This xbrl factoid is mapped to two dbf rows that might have different sign conventions.
+f1_comp_balance_db,32,2004,TOTAL Other Property and Investments (Lines 18-21 and 23-31),calculated_value,other_property_and_investments,,
+f1_comp_balance_db,33,2004,CURRENT AND ACCRUED ASSETS,header,,,
+f1_comp_balance_db,34,2004,Cash and Working Funds (Non-major Only) (130),reported_value,cash_and_working_funds,,doesn't say anything about major or non major in xbrl_factoid
```

Comment:
In [the XBRL Taxonomy](https://xbrlview.ferc.gov/yeti/resources/yeti-gwt/Yeti.jsp#tax~(id~8*v~72)!con~(id~78167)!net~(a~143*l~35)!lang~(code~en)!path~(g~3709*p~0_0_0_0_2_0)!rg~(rg~4*p~1)) it does list the Comparative Balance Sheet labels as "Cash and Working Funds (Non-major Only) (130)" which matches the DBF `row_literal` perfectly.

Reply:
Ok I'll just remove the note then.